src/GitHubUtil.js

```diff
@@ -13,16 +13,50 @@ const logApiResult = Log.logApiResult;
 
 const GitHubAuthentication = { type: 'token', username: Config.githubUserLogin(), token: Config.githubToken() };
 
+function commonAppender(allPages, aPage) {
```

Comment:
s/common/default/ so that the reader does not misinterpret "common" as "shared by all".

Reply:
Done.